docs/UserGuide.md

```diff
@@ -225,17 +225,20 @@ In case you accidentally make a mistake using the <md>`owe`</md> command, you ca
 
 Updates the amount of tuition fee paid by the student and the amount of tuition fee owed by the student.
 
-Format: `settle INDEX amount/AMOUNT`
+**Format:** `settle INDEX amount/AMOUNT`
 
-Examples:<br>`settle 1 amount/500.00`
+**Examples:**
+* `settle 1 amount/500.00`
+
+**Output:**
 
-![settleResult.png](images%2FsettleResult.png)
+![settleResult.jpg](images%2FsettleResult.jpg)
 
 <box type="important" header="#### Constraints">
```

Comment:
This one need 5 # right

Reply:
Thank you for pointing that out. I have pulled the changes into this branch.